static/js/components/search/CustomSortingSelect.js

```diff
@@ -1,58 +1,85 @@
+// @flow
 import React from 'react';
-import ReactDOM from 'react-dom';
-import { Select } from 'searchkit';
-import block from 'bem-cn';
-
-import type { Event } from '../../flow/eventType';
-
-export default class CustomSortingSelect extends Select {
-  showDropdown(sortSelectField: string, clickEvent: Event): void {
-    clickEvent.preventDefault();
-    let event: Event = document.createEvent('MouseEvents');
-    event.initMouseEvent('mousedown', true, true, window);
-    ReactDOM.findDOMNode(sortSelectField).dispatchEvent(event);
-  }
+import Grid, { Cell } from 'react-mdl/lib/Grid';
 
-  renderOptions(
-    translate: Function,
-    showCount: number,
-    countFormatter: Function,
-    items: Array<Object>
-  ): Array<React$Element<*>> {
-    return items.map(item => {
-      let text = translate(item.label || item.title || item.key);
-      if (showCount && item.docCount !== undefined) {
-        text += ` (${countFormatter(item.docCount)})`;
-      }
-
-      return <option key={item.key} value={item.key} disabled={item.disabled}>
-        { text }
-      </option>;
-    });
-  }
+import type { SearchSortItem } from '../../flow/searchTypes';
+
+export default class CustomSortingSelect extends React.Component {
```

Comment:
this is no longer a select box, so it should be called something like `CustomSortingColumnHeaders`